pylearn2/devtools/tests/test_format.py

```diff
@@ -556,6 +556,7 @@
     'scripts/datasets/make_cifar10_gcn_whitened.py',
     'scripts/datasets/make_cifar100_patches.py',
     'scripts/datasets/make_cifar100_gcn_whitened.py',
+    'scripts/datasets/make_svhn_pytables.py',
```

Comment:
Why did you need to add it to the whitelist? We are not supposed to add new files in there.